### DISCUSSION TITLE
Choose face (for using LOra)

### DISCUSSION THREAD

**Discussion by LucianoDaluz** [Bing-su/adetailer#533] [Ideas]
I just found out if you use a Lora on the Detailer prompt it gives great results, but it apply the restoration with all the faces, what if we could choose the face we want the detailer to work on? It would even be great when yu have multiple faces and you just want to apply the tool in the front faces! Please!

**Comment by zethfoxster** [comment]
You already can. Go in to your settings in the adetailer settings... Choose from left to right mode.. And then use [SEP] token ... " Bobby <The Bobby Lora: 1> [SEP] Tracy <The Tracy Lora: 1>" ..
One thing I like to do is generate a few images until I find one that would have Bobby on the left side and Tracy on the right side... And then I recycle the seed of the one that I picked.. and then turn on a detailer so it uses the Lora on the correct people... And then if you only want the biggest faces which are probably the subjects of the picture to be detailed ... Set the "keep k largest" to the amount of people that you're going to fix up.. in this case with Bobby and Tracy I would set that to keep k largest 2.. that makes it only keep the two largest mask that it detected.
